### ISSUE TITLE
Accessing Automation accounts in different tenants and subscriptions

### ISSUE THREAD

**Issue by janegilring** [azureautomation/azure-automation-ise-addon#130]
Scenario 1: A Microsoft account (formerly "Live ID") account is delegated access to a number of subscriptions. All of these shows up as expected in the Subscriptions dropdown menu in the ISE Add-on after signing in.

Scenario 2: An Azure AD account in tenant A has been delegated co-owner permissions to an Azure subscription in tenant B. This subscription does not show up in the Subscriptions dropdown menu in the ISE Add-in after signing in. Only the subscriptions in tenant A is shown.

Is there any known issues which prevents scenario 2 from working as expected?

**Comment by eamonoreilly** [comment]
Hi Jan, this should work. Can I confirm you are running the latest version of the ISE AddOn (0.2.3.7)? I'll try and reproduce the behaviour but I tried the following and it worked for me.
I have a subscription I created under a Microsoft account (Hotmail.com). I then invited myself that has a @Microsoft.com tenant as an admin on this subscription and when I log in with my @Microsoft.com id I am able to access the subscription. Is this the scenario you are trying or something different?
Thanks,
Eamon
